test/bus.test.js

```diff
@@ -10,17 +10,34 @@ let token;
 describe('Crud operation for buses ', () => {
 	before((done) => {
 		db.users.destroy({
-			truncate: true,
-			restartIdentity: true,
+			truncate : true, 
+			cascade: true ,
+			restartIdentity: true			
 		});
 		done();
 	});
 	beforeEach(async () => {
+		await db.operators.destroy({
+			truncate : true, 
+			cascade: true ,
+			restartIdentity: true,
+		});
+		await db.drivers.destroy({
+			truncate : true, 
+			cascade: true ,
+			restartIdentity: true,
+		});
+		await db.users.destroy({
+			truncate : true, 
+			cascade: true ,
+			restartIdentity: true			
+		});
 		const user = {
 			fullname: 'cyifuzo jean damascene',
 			username: 'cyifuzo',
 			email: 'operator@andela.com',
 			role: 'client',
+			userType: 'Operator',
 			password: 'test123',
 		};
```

Comment:
create a dummy data file and add in this user object,
everywhere you're using this object should be called/imported from the dummy data file